src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag

```diff
@@ -31,7 +31,14 @@
   <div id="commentBar-${divId}" class="row">
     <div class="col-xs-10">
     <span class="text-muted">
-      From: ${fn:escapeXml(frc.commentGiverName)} [${frc.createdAt}] ${frc.editedAt}
+      <c:choose>
+        <c:when test="${frc.commentFromFeedbackParticipant}">
+          Comment from response giver.
```

Comment:
I made this change because in results page of student, response giver can be anonymous, so it shouldn't be revealed by displaying comment giver. If it looks good, I'll make the god mode changes.